test/e2e/suite/conformance/certificates/venafi/venafi.go

```diff
@@ -38,6 +38,10 @@ var _ = framework.ConformanceDescribe("Certificates", func() {
 		// key or using the same private key multiple times.
 		certificates.ECDSAFeature,
 		certificates.ReusePrivateKeyFeature,
+
+		// Venafi does not support signing a certificate with only URISANs and an
+		// empty common name.
+		certificates.URISANsFeature,
```

Comment:
I think it might supports this now you've fixed the FriendlyName issue?

Reply:
Ah yes, I think that's true